### ISSUE TITLE
New ROM picker doesn't find zipped ROMs

### ISSUE THREAD

**Issue by henderjj** [davidblackuk/MountFuji#21]
Zipped ROMs work fine with Hatari, so most of mine are still zipped. In the ROM selector, I only see the unzipped ROMs (.img files).

Browsing for the zipped ROM still works fine, so it's a very minor issue.

**Comment by davidblackuk** [comment]
Hadn't thought of zipped ROMs. Is it always one rom per zip? Fuji has to scan the ROM image to get the version info and country code. If it's only one file per zip then it should be relatively efficient to scan the zip content.

**Comment by henderjj** [comment]
Yes, it's just one ROM per zip file. You just use the zip file directly with Hatari in the same way that you'd use the img file. I don't think Hatari would know which ROM to use if there were multiple files in the zip.

________________________________
